src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -97,6 +98,18 @@ private static String getTestFilePath(String stem, String suffix) {
         .toString();
   }
 
+  // helper function test cvr conversion routine
+  private static void runDominionCvrConversionTest(String stem) {
+    String dominionDataFolder = getTestFilePath(stem, "_cvr_export");
+    List<String> filesWritten = TabulatorSession
+        .convertDominionCvrJsonToGenericCsv(dominionDataFolder);
+    for (String convertedFile : filesWritten) {
+      String referencePath =
+          convertedFile.substring(0, convertedFile.length() - 4) + "_expected.csv";
```

Comment:
Is there a more robust way to get the part of a file before the extension? This would break if the length of the extension changed, right?

Reply:
e.g. you could write a regex or use something like this: https://stackoverflow.com/questions/924394/how-to-get-the-filename-without-the-extension-in-java

Reply:
Regex is notoriously slow and has some other issues that cause us to avoid using it at Google, but I do agree that we should use a more robust method here. Maybe splitting on `.`? I guess that could fail if there are .s in the folder names (heaven forfend!). Maybe the apache.commons solution would work well, since we're already pulling that library in in `ResultsWriter`?